examples/demo.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from etsdemo.main import main
+
+HERE = os.path.dirname(__file__)
+
+if __name__ == "__main__":
+    main(
+        [
+            {
+                "version": 1,
+                "name": "TraitsUI Demo",
+                "root": os.path.realpath(os.path.join(HERE, "demo")),
+            },
+        ]
+    )
```

Comment:
This file replaces the previous `demo.py`.  But I am not sure we actually need to resurrect this file:
(1) This script requires `etsdemo` to have been installed.
(2) The content of the demo requires `traitsui` to have been installed.

With that, running `etsdemo` from the command line will pull up the same content.

The only difference this script will make is that it will only show TraitsUI's example. Whereas launching `etsdemo` might show `traits`, `pyface`, `chaco`, ... examples if these other projects have also added the corresponding entry point for `etsdemo`.

Since this script requires maintenance (making sure the path is right etc), perhaps we should remove this script and add an instruction to "run `etsdemo`" in the README.  If we really need to have a script to exclude examples from other packages, it is easy to add this script back.

Reply:
let's go with the option to remove this script in it's entirety and document the use of `etsdemo` instead